Common/wvDemopi.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Postinstall for OSEHRA VistA Plan 6
+# See IMPORTANT notes in the .m file.
```

Comment:
What important notes are these? This is the only mention of IMPORTANT in the change set.